ssh/ssh2ws.go

```diff
@@ -50,6 +50,22 @@ func HandlerWebsocket(ws *websocket.Conn) {
 		}
 
 		key, err := apiClient.GetPublicKey(fingerprint, device.TenantID)
+		// TODO This is just a draft
+		username := strings.Split(user, "@")[0] // username without '@'
```

Comment:
Here, I thought It could check if `username` is equal to `*`, so all `usernames` would be enabled.